src/Android/Xamarin.Android/Samples/Network Analysis/RouteAroundBarriers/readme.md

```diff
@@ -0,0 +1,66 @@
+# Route around barriers
+
+Find a route that reaches all stops without crossing any barriers.
+
+![Screenshot](RouteAroundBarriers.jpg)
+
+## Use case
+
+You can define barriers to avoid unsafe areas, for example flooded roads, when planning the most efficient route to evacuate a hurricane zone. You could also use this functionality to plan routes when you know an area will be inaccessible due to a community activity like a run or a market night.
```

Comment:
Maybe a little more context?
"When solving a route, barriers allow you to define portions of the network that cannot be traversed. ..."